source/_integrations/calendar.markdown

```diff
@@ -180,3 +180,16 @@ data:
   start_date: "2022-10-01"
   end_date: "2022-10-02"
 ```
+
+Recent changes to Calendars in Home Assistant mean that zero duration Calendar events
+are no longer allowed. The following would create a one minute long event starting "now".
+
+```yaml
+service: calendar.create_event
+target:
+  entity_id: calendar.device_automation_schedules
+data:
+  summary: "Example"
+  start_date_time: "{{ now() }}"
+  end_date_time: "{{ now() + timedelta(minutes=1) }}"
```

Comment:
What would be the use-case for this?

Reply:
Hi Franck,
I use this to create an event in a Calendar when my UPS does a self-test.